examples/pytorch/mnist/main.py

```diff
@@ -214,7 +211,7 @@ def main():
         scheduler.step()
 
     if args.use_fp8_infer:
-        calibrate(model, device, test_loader)
+        calibrate(model, device, test_loader, args.use_fp8)
```

Comment:
Since calibration is not necessary for a model trained in fp8, another approach would be:
```suggestion
    if args.use_fp8_infer and not args.use_fp8:
        calibrate(model, device, test_loader, args.use_fp8)
```

Reply:
Done/tested